integration/spark/src/test/scala/org/apache/carbondata/spark/util/ExternalColumnDictionaryTestCase.scala

```diff
@@ -205,6 +205,20 @@ class ExternalColumnDictionaryTestCase extends QueryTest with BeforeAndAfterAll
       loadSqlRelation, "deviceInformationId", "10086")
   }
 
+  test("COLUMNDICT and ALL_DICTIONARY_PATH can not be used together") {
+    try {
+      sql(s"""
+        LOAD DATA LOCAL INPATH "$complexFilePath1" INTO TABLE loadSqlTest
+        OPTIONS('COLUMNDICT'='$extColDictFilePath1',"ALL_DICTIONARY_PATH"='$extColDictFilePath1')
+        """)
+      assert(false)
```

Comment:
assert(false) is not required as Load command will throw an exception and control will go to exception block